documentation/source/getting_started/03_quickstart.rst

```diff
@@ -0,0 +1,89 @@
+===========
+Quick Start
+===========
+
+Relations
+---------
+
+The relations created by ``rnlp`` include the following:
+
+  * Sentence’s Relative Position in Block:
+
+    * ``earlySentenceInBlock``: Sentence occurs within the first third of a block’s length.
+    * ``midWaySentenceInBlock``: Sentence occurs between the first and last third of a block’s length.
+    * ``lateSentenceInBlock``: Sentence occurs within the last third of a block’s length.
+
+  * Word’s Relative Position in Sentence:
+
+    * ``earlyWordInSentence``: Word occurs within the first third of a sentence.
+    * ``midWayWordInSentence``: Word occurs between a third and two-thirds of a sentence.
+    * ``lateWordInSentence``: Word occurs within the last third of a sentence.
+
+  * Relative Position Between Items:
+
+    * ``nextWordInSentence``: Pointer from a word to its neighbor.
+    * ``nextSentenceInBlock``: Pointer from a sentence to its neighbor.
+
+  * Existential Semantics:
+
+    * ``sentenceInBlock``: Sentence occurs in a particular block.
+    * ``wordInSentence``: Word occurs in a particular sentence.
+
+  * Low-Level Information about words:
+
+    * ``wordString``: A string representation of a word.
+    * ``partOfSpeechTag``: The word’s part of speech.
```

Comment:
How about "The word's part of speech" (POS), determined by the NLTK POS tagger.

Reply:
Good point, let me add that in first thing on Hackathon this Friday.

Reply:
Resolved in the commit e5a6607e8558dd695f076778231d0ab6b34cfbfe